sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -833,33 +832,112 @@ BOOT_POLICY_BAD_MANIFEST_CASES = [
     ],
 ) for t in BOOT_POLICY_BAD_MANIFEST_CASES for slot, offset in SLOTS.items()]
 
-[opentitan_functest(
-    name = "boot_policy_bad_manifest_{}_{}".format(
-        t["name"],
-        slot,
-    ),
-    cw310 = cw310_params(
-        exit_success = t["exit_success"],
-    ),
-    key = "multislot",
-    ot_flash_binary = "boot_policy_bad_manifest_{}_{}_img".format(
-        t["name"],
-        slot,
-    ),
-    targets = [
-        "cw310_rom",
-        "dv",
-        "verilator",
-    ],
-) for t in BOOT_POLICY_BAD_MANIFEST_CASES for slot in SLOTS]
+[
+    otp_json(
+        name = "otp_json_sec_ver_{}".format(sec_ver),
+        partitions = [
+            otp_partition(
+                name = "CREATOR_SW_CFG",
+                items = {
+                    "CREATOR_SW_CFG_MIN_SEC_VER_ROM_EXT": "{}".format(sec_ver),
```

Comment:
Nit: I think you can just use the int without stringifying it

Reply:
No, values have to be strings, otherwise `_parse_hex()` errors out.